src/lib/intercom.ts

```diff
@@ -1,19 +1,13 @@
-// Settings for boot method (camelCase as per react-use-intercom docs)
-export const INTERCOM_BOOT_PROPS = {
-  name: 'Zephyr Cloud',
+import { IntercomBootProps } from 'react-use-intercom';
+
+export const INTERCOM_BOOT_PROPS: IntercomBootProps = {
+  appId: "xyxkmxlj",
```

Comment:
Are you sure about hardcoding the appId?

Reply:
It's already hardcoded on the `<InterecomProvider>` I added it here because the type prop requires it.